packages/components/src/components/domainproperties/issues/models.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Draft, immerable, produce} from 'immer';
+import { DomainDesign } from '../models';
+import { Record } from 'immutable';
+
+export class IssuesListDefModel {
```

Comment:
I forgot to mention this in the initial CR, we should add jest tests for the various functions of this IssuesListDefModel (see examples in `packages/components/src/components/domainproperties/dataclasses/models.spec.ts`).

Reply:
done!